cookbooks/supermarket-deploy/libraries/helpers.rb

```diff
@@ -24,7 +24,7 @@ def server_fqdn_for(product_key)
         "#{product_key}.chef.co"
       else
         # A-U-R instances include the environment
-        "#{product_key}-#{environment}.cd.chef.co"
+        "#{product_key}-#{environment}.chef.co"
```

Comment:
Correct. The oc-id entry was actually always no `.cd`, so it's actually been broken.